src/components/WwwFrame/TheHeader.vue

```diff
@@ -748,8 +786,19 @@ export default {
 			hasLentBefore: this.cookieStore.get(hasLentBeforeCookie) === 'true',
 			hasDepositBefore: this.cookieStore.get(hasDepositBeforeCookie) === 'true',
 		});
+		window.addEventListener('resize', _throttle(() => {
+			this.determineIfMobile();
+		}, 200));
+	},
+	beforeDestroy() {
+		window.removeEventListener('resize', _throttle(() => {
```

Comment:
This probably needs to use a method like in the other PR, rather than the throttle (unless we check that the throttle will be identified correctly as the same method, which it might, it depends on its implementation).

Reply:
+1 Put the `_throttle` call inside the `determineIfMobile` method instead, then just that method needs to be added/removed as the event listener.